src/pages/AdminManageInstitutions.js

```diff
@@ -118,22 +145,6 @@ export default function AdminManageInstitutions(props) {
         onOKRequest: modalSave,
         onCloseRequest: closeAddInstitutionModal
       }),
-      div({ style: {display: 'flex', justifyContent: 'space-between'}}, [
-        div({className: 'left-header-section', style: Styles.LEFT_HEADER_SECTION}, []),
-        div({className: 'right-header-section', style: {
-          display: 'flex',
-          alignItems: 'flex-end'
-        }}, [
-          a({
-            id: 'btn_addInstitution',
-            className: 'btn-primary btn-add common-background',
-            onClick: addInstitution
-          }, [
-            div({className: 'all-icons add-dac_white'}),
-            span({}, ['Add Institution'])
-          ])
-        ])
-      ])
     ])
   );
-}
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9985188)
```suggestion
}
```